schema_utils_test.py

```diff
@@ -273,11 +273,6 @@ def validate_schema(schema):
         _validate_ui_config(
             schema[SCHEMA_KEY_TYPE], schema[SCHEMA_KEY_UI_CONFIG])
 
-    if SCHEMA_KEY_CHOICES in schema and SCHEMA_KEY_POST_NORMALIZERS in schema:
```

Comment:
These lines are in the test file and are untested.

Reply:
> These lines are in the test file and are untested.

Sorry, I'm unable to understand this, can you point which test you are referencing here?

Reply:
Oh, I see now! (Marking resolved)